commcare_connect/users/migrations/0027_remove_connectiduserlink_connect_user_and_more.py

```
@@ -0,0 +1,23 @@
+    operations = [
+        migrations.RemoveConstraint(
+            model_name="connectiduserlink",
+            name="connect_user",
+        ),
+        migrations.AddConstraint(
+            model_name="connectiduserlink",
+            constraint=models.UniqueConstraint(
+                fields=("user", "commcare_username", "hq_server"), name="connect_user"
+            ),
+        ),
+    ]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)